data/pilots/rebel-alliance/t-65-x-wing.json

```diff
@@ -307,6 +307,262 @@
       "extended": true,
       "keywords": ["X-wing"],
       "epic": true
+    },
```

Comment:
Pilots seems to be alphabetically sorted in their respective chassis. I was considering making another section entirely of standardpilots or something, but I'm worried about compatibility with whatever xws parsers people may have already written.